core/src/main/java/com/gentics/mesh/changelog/highlevel/HighLevelChangesList.java

```diff
@@ -16,14 +16,14 @@
 @Singleton
 public class HighLevelChangesList {
 
-	@Inject
-	public ExtractPlainText plainText;
+	protected final ExtractPlainText plainText;
```

Comment:
Hm, shouldn't it be vice versa - the field injections **don't** care about order, but the constructor params injections **do** care?

Reply:
The problem with the field injections is that dagger will create Factory classes that have methods for actually injecting the fields and the order of the parameters of this method is not necessarily the order of the fields in the class.

By using a constructor, the order of the parameters is well defined.